frontend/doc-site/05__Annotate and Analyze Your Data/5_4__Annotating Data.mdx

```diff
@@ -74,7 +74,7 @@ anndata.obs = anndata.obs.join(new_annotations)
 
 As described in the [hosted tutorials](/docs/04__Analyze%20Public%20Data/4_1__Hosted%20Tutorials), we do not officially support hosted or multi-user use of cellxgene. However, we recognize that the app is often adapted for this purpose, and have tried to provide a "safe path" for multi-user setups that avoids overwriting data.
 
-Specifying a single file name for multiple contributors will result in data overwriting. To avoid this, you can instead specify an output directory and allow cellxgene to assign filenames.
+Specifying a single file name for multiple contributors will result in data overwriting. To avoid this, you can instead specify an output directory and allow CELLxGENE to assign filenames.
```

Comment:
```suggestion
Specifying a single file name for multiple contributors will result in data overwriting. To avoid this, you can instead specify an output directory and allow CELLxGENE Annotate to assign filenames.
```